src/ILVerify/src/Program.cs

```diff
@@ -199,13 +228,22 @@ private int Run(string[] args)
             _typeSystemContext.InputFilePaths = _inputFilePaths;
             _typeSystemContext.ReferenceFilePaths = _referenceFilePaths;
 
-            _typeSystemContext.SetSystemModule(_typeSystemContext.GetModuleForSimpleName("mscorlib"));
+            EcmaModule systemModule = _typeSystemContext.GetModuleForSimpleName(SystemModuleSimpleName);
+            _typeSystemContext.SetSystemModule(systemModule);
 
             foreach (var inputPath in _inputFilePaths.Values)
             {
                 _numErrors = 0;
 
-                VerifyModule(_typeSystemContext.GetModuleFromPath(inputPath));
+                EcmaModule module;
+                // special case for verifying mscorlib, which was already loaded
+                string simpleName = Path.GetFileNameWithoutExtension(inputPath);
+                if (simpleName == SystemModuleSimpleName)
```

Comment:
It would be better to instead fix GetModulePath to work on already loaded modules.

The implementation of the TypeSystemContext in the compiler works that way:
https://github.com/dotnet/corert/blob/1896a8cab2f6f83a86ba3879a02018230c549fcf/src/ILCompiler.Compiler/src/Compiler/CompilerTypeSystemContext.cs#L155